### DISCUSSION TITLE
Fosjsrouting file not loaded after Minor Update

### DISCUSSION THREAD

**Discussion by idna001** [pimcore/pimcore#16870] [Q&A / Support] [answered]
Hi, 

after upgrading Pimcore 11 with any minor update via 'composer update', I often get this error message, see below.
The frontend is working, but I can't log into the Admin Backend anymore. 
I have to enable debug Mode, then it's suddenly using another file.  How can I regenerate that specific file?

I already cleared Pimcore cache and built the 'vendor' and 'var' folder new.

![grafik](https://github.com/pimcore/pimcore/assets/46223359/78371fdb-f0e8-4ac4-84cf-c3cc9935fd65)

Thanks for a solution or a workaround
Andreas

**Comment by skoch98** [comment]
Hi @idna001, did you already try to regenerate the FOS file?

Run `php bin/console fos:js-routing:dump`, this will output:
```
Dumping exposed routes.

[dir+]  /var/www/html/public/js
[file+] /var/www/html/public/js/fos_js_routes.js
```

BR,
Sebastian

**Reply by idna001** [reply]
Hey Sebastian, 
thank you for the answer. 
I tried your command, and I can confirm your output.

But it did not solve my problem: 
My backend in this installation is still only reachable via Debug Mode turned on.

Btw, the point where it stops working is after login into the backend. Frontend works completely. 
Any idea?

**Reply by skoch98** [reply]
In the screenshot I see that you are loading several Javascript files from Chrome extensions (the ones with the CSP header errors). But that doesn't matter.

But the path where the FOS routes are expected is interesting. That doesn't correspond to the location of the output of the command.
Try to change the location of the command with by adding `--target=public/bundles/fosjsrouting/js/router.js`. Alternatively, you can move the public/js/... files to the place where it is expected.

Please delete all caches via php bin/console cache:clear and delete the JS files under /path/to/pimcore/var/admin/

And please make sure that you delete all browser caches and local-stroage items.

**Reply by idna001** [reply]
Thank you, for the tips. 
Unfortunately, that also didn't work. I have the error on two installations. One is running on nginx and the other is running on Apache. 
I assume that I have some problems with .htaccess config, the symlinks obviously don't work. 
Or the regeneration of the file doesn't work. 

If DEBUG Mode is turned on,   I can access the backend, but with very limited features. The console also shows a lot of errors. 

I will try further. If I find a solution or the reason for my problem, I will report it.
Otherwise, I have to reinstall my installation..

**Comment by idna001** [comment, marked as answer]
Hello, 
with the assistance of @skoch98 we were able to fix the error:

The problem came from some conflicts with the assets mapping. 
It can be solved with the following steps: 

1. install assets new
2. compile asset-map
3. composer install
4. migrations:migrate
5. clearing cache
6. building a new vendor folder.

Thanks for the great support!
Andreas

**Reply by skoch98** [reply]
Important context information:
In this Pimcore installation the Symfony asset mapper bundle was used to handle custom UI stuff.

So the issue would not occur on a plain Pimcore Demo / or normal installation.
